src/webhook/webhook.service.ts

```diff
@@ -0,0 +1,99 @@
+import { HttpService } from '@nestjs/axios';
+import {
+  ForbiddenException,
+  Injectable,
+  NotFoundException,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { CreateWebhookRequest, FetchedWebhook, WebhookEvent } from './dto';
+import { WebhookRepository } from './webhook.repository';
+import { Recipe } from '@prisma/client';
+import Cryptr from 'cryptr';
+
+@Injectable()
+export class WebhookService {
+  constructor(
+    private readonly httpService: HttpService,
+    private readonly webhookRepository: WebhookRepository,
+  ) {}
+  cryptr = new Cryptr(process.env.CRYPTR_SECRET_KEY);
```

Comment:
Didn't know about this library - I see it's an easy to use library to use `aes-256-gcm` algorithm.

Normally I would say - ok 👍🏻 - but because we are learning, I would like you to use a built-in `node:crypto` library https://nodejs.org/api/crypto.html which can be used for AES and more. Hint #1: you might end up with an "util" class for this as there will be some code here. Hint #2: AES-256-GCM is a good choice 👍🏻 Don't use AES-256-CBC.